pr_4/javascript/random_samples/cross_site_request_forgery.js

```
@@ -0,0 +1,32 @@
+{
+    app.get("/", (req, res) => {
+        // Noncompliant: `sameSite` is set to 'none'.
+        res.cookie('cookieName', 'cookieValue', { sameSite: 'none', secure: true })
+        res.render("index.html")
+    })
+}
+// {/ex-fact}
+
+
+// {fact rule=cross-site-request-forgery@v1.0 defects=0}
+var express = require('express')
```

Biome (2.1.2)

[error] Shouldn't redeclare 'express'. Consider to delete it or rename it.

'express' is defined here:


(lint/suspicious/noRedeclare)